### ISSUE TITLE
Missing translations

### ISSUE THREAD

**Issue by prschopf** [ampache/ampache#1738]
# Environment

 * Ampache version: 3.8.8

# Description

I couldnt find these strings in https://www.transifex.com/ampache/ampache/dashboard:

![trad1](https://user-images.githubusercontent.com/16743256/41543716-cb163f8a-72ed-11e8-89da-41e26b677cf9.jpg)
![trad2](https://user-images.githubusercontent.com/16743256/41543718-cb3fdf48-72ed-11e8-87a0-d44f56174623.jpg)
![trad3](https://user-images.githubusercontent.com/16743256/41543719-cb658c8e-72ed-11e8-8662-9cef8ba6c740.jpg)

How to translate them?

**Comment by Psy-Virus** [comment]
Hi @prschopf,

"Open Basedir" is basically a PHP setting's/module's name like Zlib, Gettext, iconv and so on.
The others are, like the option descriptions, categories and subcategories, saved within the preference table of the database. But they are lowercase written there. So they definitely will be processed somehow.
I was digging a bit. But sometimes the Ampache code makes it kinda hard for me to find, what I'm looking for :D

But I guess to fix this, we need to modify the `gather-messages.sh` in `locale/base` dir.
A while ago I built in a workaround to also translate the preference descriptions from the database that were untranslated before. That's what the `translatable-database-strings.txt` in the same directory is for.

I'll see what I can do. Should be easy to fix once I get what I did there :-P

:-)
